### DISCUSSION TITLE
`nvim-notify` shows "No information available" when LSP is working.

### DISCUSSION THREAD

**Discussion by hopezh** [LazyVim/LazyVim#4577] [Q&A] [answered]
I don't know why `nvim-notify` keeps telling me `No information available` when I use `K`(hover) to show definition of a module in `python` when `LSP` for `python` is working correctly in the current buffer, as show below. 

<img width="1500" alt="Screenshot 2024-10-12 at 11 20 50 PM" src="https://github.com/user-attachments/assets/021461d9-9dc8-4d01-8393-e7f5c9972d63">

<img width="1500" alt="Screenshot 2024-10-12 at 11 25 55 PM" src="https://github.com/user-attachments/assets/b74cd87e-bcce-4e9c-8ee3-c912c5d84243">

My neovim config is [here](https://github.com/hopezh/oatvim08). 

My `nvim-notify` config is: 

```lua
return {
    "rcarriga/nvim-notify",

    opts = {
        -- when using transparent in tokyonight, need to set a bg color here
        background_colour = "#000000",

        -- move message window to lower right corner
        top_down = false,
    },
}
```

**Comment by dpetka2001** [comment, marked as answer]
It's because you have enabled `pyright` and `ruff` by yourself and didn't enable the Python Extra. By enabling the Python Extra, ruff is configured to not provide `hoverProvider` capability. By not enabling the Python Extra you have to configure it yourself otherwise Hover information is probed by every LSP server available. 

So, you can either enable the Python Extra or you can configure `noice.nvim` to not show this kind of notification. If you decide to go the `noice.nvim` configuration route, you can see the available settings for `hover` [here](https://github.com/folke/noice.nvim/blob/df448c649ef6bc5a6a633a44f2ad0ed8d4442499/README.md?plain=1#L231-L237).
